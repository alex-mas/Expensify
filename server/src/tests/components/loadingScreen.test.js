import React from 'react';
import toJSON from 'enzyme-to-json';
import {shallow} from 'enzyme';
import LoadingScreen from '../../components/loadingScreen.js';


describe('ErrorNotFound component unit tests', function(){
    test('should render ErrorNotFound correctly',()=>{
        const wrapper = shallow(<LoadingScreen/>);
        expect(wrapper).toMatchSnapshot();
    });
});
