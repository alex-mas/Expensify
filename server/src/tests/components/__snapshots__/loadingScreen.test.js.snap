// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorNotFound component unit tests should render ErrorNotFound correctly 1`] = `
<div
  className="loader"
>
  <img
    className="loader__image"
    src="/images/loader.gif"
  />
</div>
`;
